### ISSUE TITLE
Start menu jumps down when start typing into search bar

### ISSUE THREAD

**Issue by gregoriusus** [valinet/ExplorerPatcher#1803]
I am using taskbar on the left side of monitor vertically. Start button is on upper left corner of monitor. When I click start button it opens start menu on upper left corner of the monitor. So far, everything ok. Then, when I click into search bar in start menu, start menu jumps down on the left downs corner of the monitor. Otherwise, functionality is ok, but this jump is unpleasant for eyes.

**Comment by gregoriusus** [comment]
It looks like when one clicks in search bar. Start menu is positioning as taskbar is horizontally

**Comment by gregoriusus** [comment]
Like this setting are reseted, when one clicks in search bar.
<img width="365" alt="image" src="https://github.com/valinet/ExplorerPatcher/assets/4267743/d915242f-1b5e-4547-ad7a-08ec9572c26c">

**Comment by max1c** [comment]
Had this issue for months.

**Comment by Girofox** [comment]
Is this the same issue when you start typing and then the search bar collapses itself?

**Comment by gregoriusus** [comment]
It doesn't collapse, just whole Start menu jumps down as that taskbar bar is on bottom not on the left or right side of monitor

**Comment by pyrates999** [comment]
Please post windows version and build.
Please post explorer patcher version.

**Comment by Girofox** [comment]
Can you record your screen with Win + G and upload it here? Maybe we have the same issue.

**Comment by gregoriusus** [comment]
Here is a recording.
[Link](https://www.awesomescreenshot.com/video/19872065?key=93a92c19ebdcedc15681c462f64f14c7)

This is only glitch that this patch has. Otherwise it works great

**Comment by Mystickone** [comment]
I am having the same issue with the taskbar at the top.  Even when clicking directly on the 'Magnifying Glass' icon, the search window pops up as if the taskbar was positioned at the bottom, as previously mentioned.

**Comment by pyrates999** [comment]
You still haven't told us what build of windows you're running.

**Comment by Tyler-IN** [comment]
Happens on 10.0.22621.2134

**Comment by gregoriusus** [comment]
Edition	Windows 11 Pro
Version	22H2
OS build	22621.2134
Experience	Windows Feature Experience Pack 1000.22659.1000.0

**Comment by pyrates999** [comment]
Please wait for this to get fixed then.

**Comment by Nestoro** [comment]
Still present in 22621.2361.58.1 with build 22621.2283

looking forward to getting this fixed

**Comment by TheBatman09** [comment]
Are you guys aware of a previous patch that didn't have this issue ? I'm not really sure based on the comments 😅

**Comment by Tyler-IN** [comment]
Yeah, not sure when it started, but it was only a release or two before 22621.1992.56.1

**Comment by gregoriusus** [comment]
Were there a patch that didn't have this issue?

**Comment by TheBatman09** [comment]
Well I just tried [22621.1413.54.4](https://github.com/valinet/ExplorerPatcher/releases/tag/22621.1413.54.4_8e9403c) but it had the issue too, maybe some borken configuration stays even if you rollback

**Comment by gregoriusus** [comment]
Well, I think this issue is still present. Never has been fixed.

**Comment by gregoriusus** [comment]
Any progress? With this fixed, it would be perfect Windows 11 patch.

**Comment by davie2000** [comment]
I really hate the new windows 11 taskbar - the more I love your software!
Thanks a lot!!!

But the above described behaviour is really disappointing.
Wish I could vote this bug up for quicker fixing.

**Comment by luxifr** [comment]
I just found this tool today and outside of this here issue it's a 10/10... @valinet I'd be happy to pay a bounty for fixing this... nothing crazy, but certainly more than 2 cups of coffee :D

**Comment by bllngr** [comment]
Facing the same issue.

ExplorerPatcher 22621.3527.65.5
Windows 11 Version 10.0.22621 Build 22621

**Comment by pyrates999** [comment]
your build number for windows is incomplete.  Please run winver to show it.

**Comment by hizperion** [comment]
still happening in 

Windows 11 Version 23H2 Build 22631.3737


ExplorerPatcher 22621.3527.65.5

**Comment by Cookie4Free** [comment]
can confirm. it's still happenning.

Windows 11 Version 23H2 Build 22631.3880
ExplorerPatcher 22621.3527.65.5

**Comment by blkallcaps** [comment]
+1 experiencing this

- Windows 11 Pro Version 23H2 OS Build 22631.4317
- ExplorerPatcher 22621.3880.66.6

**Comment by shamlord** [comment]
Also experiencing this. 

Windows 11 Pro 24H2, build 26100.2605
Explorer Patcher version 22621.4317.67.1

**Comment by nesherman** [comment]
Another person experiencing this.

Windows 11 IoT Enterprise LTSC 24H2, 26100.2894
Explorer Patcher version 22621.4317.67.1

**Comment by Mohsyn** [comment]
Same Issue on 
Windows 11 Pro - 24H2 Build 26100.3476
EP: 22621.4317.67.1

**Comment by Drax-Reborn** [comment]
Hi,

Windows 11 Pro
Version 24H2 (OS BUILD 26100.1742)

Any update on this issue?

**Comment by Milchreislucker** [comment]
Exact same issue here in 2026. Otherwise, very happy with ExplorerPatcher, it's a life saver!!!

- Win10 style Taskbar is on the right.
- Start Menu shows up correctly.
- But, Search Results show in bottom left corner in Win11 style.

ExplorerPatcher
Version 26100.8457.70.1

Windows 11 Enterprise
23H2
22631.6936

**Comment by pyrates999** [comment]
Duplicate of: https://github.com/valinet/ExplorerPatcher/issues/1803

Please close this.

**Comment by Milchreislucker** [comment]
> Duplicate of: [#1803](https://github.com/valinet/ExplorerPatcher/issues/1803)
> 
> Please close this.

This is #1803, no?

**Comment by pyrates999** [comment]
oh sorry :) yes it is 😆
